src/lib/OpenEXRCore/openexr_priv_structs.h

```diff
@@ -83,7 +112,7 @@ typedef struct _priv_exr_file_t
     exr_read_func_ptr_t  read_fn;
     exr_write_func_ptr_t write_fn;
 
-    atomic_long file_offset; /**< used when writing */
+    atomic_llong file_offset; /**< used when writing */
```

Comment:
Since the explicitly sized types are available, I'd like to request we use them instead, moving off of the squishy types int, size_t, short, long, and long long. By squishy I mean sometimes compilers agree on the size, and sometimes they don't; across vendors, and across processors... If we use atomic_int64 here instead, there'll never be odd regressions on things like ARM32 as we've seen in the main branch recently.

Reply:
well, the problem is, these types are not really available on msvc - they still haven't implemented c11, and most of their file types are all still 32-bit, it's kind of ... surprising. But sure, we can do that

Reply:
MSVC 16.8 preview 3 introduced c11 support at long last (Sept 2020). I wonder if the Windows + OpenEXR crowd is on latest compilers or not. Personally, I reflexively upgrade MSVC because I'm crazy and to make the constant "upgrade now!" pop ups disappear. Not sure if that's par for the course though.

Reply:
our CI config seems to be using 16.4, I'd have to lean on you and Christina to manage what is available or in use on the windows side. Either way, I'll convert to atomic_int64 for clarity, once you confirm a 16.8+ version of msvc, let me know the preprocessor detection magic and we can put it in the file declaration (I just put the shortest path I needed using Interlocked* in).